Feature/ch62

RaulRory opened the pull request:
Foi adicionado, Query parameters, para o enpoint de logs, para melhoria dos filtros!

infofranco2005 committed:
criando branch crypt [61]

planattife committed:
[ch62]-Filtros de logs criados

mbj-junior committed:
Implementação da base64 e junção da criptografiarandomica

Commit:
Add Urrl enconde, modificando arquivo packge.json, excluindo a pasta service

planattife committed:
Mudancas para evitar erro de Unhandled Promisse

planattife committed:
Tirando comentario dos ifs

planattife commented:
Mudei o modo como nos fizemos os if`s dos filtros de logs, por que ao realizar as pesquisas com os filtros estava dando o seguinte erro: 

UnhandledPromiseRejectionWarning: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client

Commit:
resolvendo amerda que eu fiz adicionei o nodemodeles na branch

Commit:
incluindo script de cryptografica [ch61]

mbj-junior committed:
atualização do método de criptografia

mbj-junior committed:
cofiguração do .env

infofranco2005 committed:
banco de dados cript

infofranco2005 committed:
cripto DB [ch61]

Commit:
Config padrão Standart para code Style

Commit:
Atualizando a branch

Commit:
Atualizando branch com verificação de usuario

patrickzequiel commented on src/controllers/log.js:
```diff
@@ -2,23 +2,38 @@ const model = require('../models')['logs']
 
-    return result
-}
+  if(type){
```
Lindo, só seguir a orientação do Will, pois o código está dizendo que só retornará se tiver todos os params dos ifs.

Exemplo:
Players.getAll = async (req, res, next) => {
  const { nationality, score } = req.query
  let query = { where: {} }

  if (nationality) {
    query.where['nationality'] = {
      [Op.like]: `%${nationality}%`
    }
  }

  if (score) {
    query.where['score'] = {
      [Op.gte]: parseInt(score)
    }
  }

  const data = await model.findAll(query)

  res.status(200).json({
    total: data.length,
    data
  })
}

mbj-junior committed:
Delete testeCrypto.js

Arquivo usado para testar as funcionalidades de criptografia e descriptografia.

planattife committed:
melhorando os filtros com query string

planattife commented:
Conforme observado pelo Patrick, fiz a mudança nas query string para que seja possivel varios filtros serem aplicados ao mesmo tempo, para assim gerar uma pesquisa mais completa

Commit:
atualizando função para gerar o token

Commit:
Merge branch 'feature/ch61b' of https://github.com/codenation-dev/squad-4-ad-nodejs-kroton-1 into feature/ch61b

RaulRory committed:
Merge pull request #9 from codenation-dev/feature/ch61b

Feature/ch61b

RaulRory reviewed (commented):
De acordo com as modificações

Commit:
Resolvendo os conflitos V.2

Commit:
Add Documentação com a aplicação

RaulRory reviewed (commented):
Limpaando repositorios, atualizando projetos comas reviews das Tasks finalizadas!!
